packages/landingpage-new/components/shared/navigationMenu.module.scss

```diff
@@ -0,0 +1,51 @@
+@use '@styles/theme';
+
+.aside {
+  margin-left: 100px;
+}
+
+.aside > .navigationMenu {
+  border-left: 1px solid var(--ds-border, #ebecf0);
+  margin-top: var(--ds-space-400, 35px);
+  padding-block: var(--ds-space-050, 4px);
+  padding-inline: var(--ds-space-150, 12px);
+  margin-bottom: var(--ds-space-400, 32px);
+  min-width: 184px;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 80px;
+  color: theme.$n-6;
+}
+
+.navigationMenu > ul > li > a {
+  font-weight: 200;
+  color: inherit;
+}
+
+li.active:before {
+  content: '';
+  position: absolute;
+  left: -14px;
+  height: 50px;
+  border-left: 2px solid theme.$p-05;
+  top: -14px;
+}
+
+.navigationMenu .active {
+  color: theme.$p-05;
+}
+
+.navigationMenu > ul {
+  list-style-type: none;
+  padding: 0;
+}
```

Comment:
I would generally prefer a class for the `ul` element instead of using a selector like this